### PR TITLE
HdmiCec json and md files updated as per code

### PR DESCRIPTION
**Pull request by pjayar213**
Documentation enhancement for HdmiCec plugin
Jira Ticket link: https://ccp.sys.comcast.net/browse/RDK-35092

### CODE REVIEW

File: HdmiCec/HdmiCec.json [hunk 1]
```diff
@@ -130,7 +193,7 @@
             }
         },
         "setEnabled":{
-            "summary": "Enables or disables HDMI-CEC",
+            "summary": "Enables or disables HDMI-CEC.\n  \n### Event \n\n No Events",
```
Comment (apatel859) [inline HdmiCec/HdmiCec.json:196]: Enables or disables HDMI-CEC driveer

File: HdmiCec/HdmiCec.json [hunk 2]
```diff
@@ -94,8 +109,53 @@
                 ]
             }
         },
+        "getDeviceList":{
+            "summary": "Gets the number of connected source devices and system information for each device. The information includes logicalAddress,OSD name and vendor ID.\n  \n### Event \n\n No Events",
```
Comment (apatel859) [inline HdmiCec/HdmiCec.json:113]: Gets the list of number of CEC enabled devices connected and system information for each device.

File: HdmiCec/HdmiCec.json (hunk 1 above)
Comment (pjayar213) [reply HdmiCec/HdmiCec.json]: Updated changes as per your review comments
Please refer changes in #2155

File: HdmiCec/HdmiCec.json (hunk 2 above)
Comment (pjayar213) [reply HdmiCec/HdmiCec.json]: Updated changes as per your review comments
Please refer changes in #2155

**Pull request closed without merging**